main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -34,17 +34,16 @@ object FlixPackageManager {
   /**
     * Installs all the Flix dependencies for a Manifest.
     */
-  def installAll(manifest: Manifest, path: Path)(implicit out: PrintStream): Result[Unit, PackageError] = {
+  def installAll(manifest: Manifest, path: Path)(implicit out: PrintStream): Result[List[Path], PackageError] = {
```

Comment:
What does this now return? All packages are only the newly installed?

I think updating the documentation would be useful.

Reply:
A list of paths to all the files - both old and new

Reply:
Perfect. Lets update the docs to say that :)